lib/tasks/deny_missed_requests.rake

```diff
@@ -2,12 +2,16 @@ desc 'Mark missed requests as denied'
 task deny_missed_requests: :environment do
   # get all requests that began yesterday and weren't approved / denied /
   # acted upon
+  # also sends an email to the user
   missed_requests = Reservation.missed_requests
   Rails.logger.info "Found #{missed_requests.size} missed requests."
 
   missed_requests.each do |request|
     Rails.logger.info "Marking as denied:\n #{request.inspect}"
-    request.update_attributes(status: 'denied')
+    request.status = 'denied'
```

Comment:
Just to clarify this for myself, the reason that this works is the only reservations whose status is `requested` and whose start date has passed are those that have not been processed, otherwise they'd either have the status `denied` (if denied) or `reserved` (if approved). Is that correct?